Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/Views/Home/StorageBucketListing.cshtml

```diff
@@ -9,23 +11,29 @@
 <html>
 <head>
     <meta name="viewport" content="width=device-width" />
-    <title>Integreation tests - Storage Bucket Listing</title>
+    <title>Google.Apis.Auth.AspNetCore3 sample - Storage bucket list</title>
 </head>
 <body>
-    <h1>Integration tests - Storage Bucket Listing</h1>
-    <ul>
-        @foreach (var bucketName in @Model)
-        {
-            <li>@bucketName</li>
-        }
-    </ul>
+    <h2>Google.Apis.Auth.AspNetCore3 sample - Storage bucket list</h2>
     @if (@Model.Count > 0)
     {
-        <p>Test passed</p>
+        <p>
+            @User.FindFirst(ClaimTypes.GivenName)?.Value these are the Storage buckets you have
```

Comment:
Again, I'd probably put that as a separate paragraph: "Logged in user: @User.FindFirst(ClaimTypes.GivenName)?.Value"

Reply:
Yep, done everywhere.